repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -189,7 +190,46 @@ class SamlPolicyTranslationFilter @Inject()(configurationService: ConfigurationS
     * @return the issuer for the embedded assertions
     * @throws SamlPolicyException if response is invalid
     */
-  def validateResponseAndGetIssuer(document: Document): String = ???
+  def validateResponseAndGetIssuer(document: Document): String = {
+    val version = 30
+    val assertionPath = "/s2p:Response/s2:Assertion"
+    val issuerPath = "/s2p:Response//s2:Issuer"
+    val signaturePath = "/s2p:Response/s2:Assertion/sig:Signature"
```

Comment:
These three paths could also be defined as constants in the companion object so the connection between the two is clearer.